### PR TITLE
Apply code and math formatting to docs

### PR DESCRIPTION
**Pull request by eric-wieser**
Will use CI to check this looks right

### CODE REVIEW

File: clifford/cga.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-
 '''
 .. currentmodule:: clifford.cga
+
```
Comment (eric-wieser) [inline clifford/cga.py]: Chanegs in this file only to make sphinx not emit warnings - rendering is the same

File: clifford/_multivector.py [hunk 2]
```diff
@@ -970,12 +921,12 @@ def join(self, other) -> 'MultiVector':
...
     def meet(self, other, subspace=None) -> 'MultiVector':
-        r"""Returns the meet of two blades.
+        r"""The meet of two blades.
 
         Computation is done with respect to a subspace that defaults to
         the join if none is given.
-                     -1
-        M \/i N = (Mi  ) * N
+
+        :math:`M \vee_i N = M_i^{-1}N`
```
Comment (eric-wieser) [inline clifford/_multivector.py]: I have no idea what this means, but is my best guess at the intent of the ascii art

File: clifford/_multivector.py [hunk 3]
```diff
@@ -901,9 +852,9 @@ def basis(self) -> List['MultiVector']:
         return thisBasis
 
     def join(self, other) -> 'MultiVector':
-        """Returns the join of two blades.
-              .
-        J = A ^ B
+        r"""The join of two blades.
+
+        :math:`J = A \wedge B`
```
Comment (eric-wieser) [inline clifford/_multivector.py]: Is there a difference in notation between the join and wedge?

File: clifford/_multivector.py [hunk 4]
```diff
@@ -763,28 +717,26 @@ def dual(self, I=None) -> 'MultiVector':
...
 
     def gradeInvol(self) -> 'MultiVector':
-        """Returns the grade involution of the multivector.
-         *                    i
-        M  = Sum[i, dims, (-1)  <M>  ]
-                                   i
+        r"""The grade involution of the multivector.
+
+        .. math::
+            M^* = \sum_{i=0}^{\text{dims}}
```
Comment (eric-wieser) [inline clifford/_multivector.py]: `M^*` is being used for both the grade involution and `conjugate` below!

File: clifford/_multivector.py [hunk 5]
```diff
@@ -13,30 +13,27 @@
```
Comment (eric-wieser) [inline clifford/_multivector.py]: See https://external-builds.readthedocs.io/html/clifford/166/generated/clifford.MultiVector.html#clifford.MultiVector for the rendered version of this file.

File: clifford/tools/g3/__init__.py [hunk 6]
```diff
@@ -2,10 +2,10 @@
 Tools for 3DGA (g3)
 
 3DGA Tools
-==========================================================
+==========
 
 Rotation Conversion Methods
---------------------
+---------------------------
```
Comment (eric-wieser) [inline clifford/tools/g3/__init__.py]: Fixes a warning

File: clifford/tools/g3c/__init__.py [hunk 7]
```diff
@@ -1400,30 +1400,30 @@ def val_rotor_between_objects_explicit(X1, X2):
...
+            R = ( beta*K(4) - (1/(2*beta)) )*(1 + gamma1*X2*X1)/(root_mu)
+            return R
         else:
-            beta_sqrd = 1/(2*(root_mu + K[0]))
-            beta = np.sqrt(beta_sqrd)
-        R = ( beta*K(4) - (1/(2*beta)) )*(1 + gamma1*X2*X1)/(root_mu)
-        return R
-    else:
-        return (1 + gamma1*X2*X1)/(np.sqrt(abs(K[0])))
+            return (1 + gamma1*X2*X1)/(np.sqrt(abs(K[0])))
```
Comment (eric-wieser) [inline clifford/tools/g3c/__init__.py]: Indented to make sphinx happy

File: docs/issues_and_changelog.rst [hunk 8]
```diff
@@ -1,84 +1,84 @@
...
-   as is definitely not intended.  However,
+     # written        meaning            possibly intended
+     e1 << e2 ^ e1 == (e1 << e2) ^ e1 != e1 << (e2 ^ e1)
+     e1 ^ e2 | e1  == (e1 << e2) ^ e1 != e1 << (e2 ^ e1)
 
-     M = MultiVector(layout2D)
-     M << (2^e0) << e1 << (3^e01) == M == 2^e0 + 1^e1 + 3^e01
-     e0 == 1^e0
-     e1 == 1^e1
-     e01 == 1^e01
```
Comment (eric-wieser) [inline docs/issues_and_changelog.rst]: As discussed on slack, no one knew what these last two examples meant

File: clifford/_multivector.py (hunk 3 above)
Comment (hugohadfield) [reply clifford/_multivector.py]: I don't know of any good notation to differentiate between them, maybe some stuff in the end of Leo's book where he talks about them and delta product etc?

**Comment by eric-wieser** [comment]
No comments from slack, let's just put this in, it can be corrected in post

**Pull request merged**